tests/test_git.py

```diff
@@ -293,3 +293,32 @@ def test_create_tag_with_message(tmp_commitizen_project):
         assert git.get_tag_message(tag_name) == (
             tag_message if platform.system() != "Windows" else f"'{tag_message}'"
         )
+
+
+@pytest.mark.parametrize(
+    "file_path,expected_cmd",
+    [
+        (
+            "/tmp/temp file",
+            'git commit --signoff -F "/tmp/temp file"',
+        ),  # File contains spaces
+        (
+            "/tmp dir/temp file",
+            'git commit --signoff -F "/tmp dir/temp file"',
+        ),  # Path contains spaces
+        (
+            "/tmp/tempfile",
+            'git commit --signoff -F "/tmp/tempfile"',
+        ),  # Path does not contain spaces
+    ],
+)
```

Comment:
nitpick: instead of using commands. [ids](https://docs.pytest.org/en/stable/example/parametrize.html#different-options-for-test-ids) might be a even better solution

Reply:
Agree! ids is a better way to describe the test cases